File: @LV1/1.6/leftbeehind/leftbeehind.cpp
#include <cstdio>

int main() {
    int a, b;

    while (true) {
        scanf("%d %d", &a, &b);
        if (a == 0 && b == 0) {
            break;
        }

        if (a + b == 13) {
            puts("Never speak again.");
        } else if (a == b) {
            puts("Undecided.");
        } else if (a > b) {
            puts("To the convention.");
        } else {
            puts("Left beehind.");
        }
    }
}

File: @LV1/1.6/leftbeehind/leftbeehind.rb
loop do
  a, b = gets.split.map(&:to_i)

  break if a.zero? && b.zero?

  if a + b == 13
    puts 'Never speak again.'
  elsif a == b
    puts 'Undecided.'
  elsif a > b
    puts 'To the convention.'
  else
    puts 'Left beehind.'
  end
end
